cli/schemas/config-file.v1.json

```diff
@@ -355,6 +355,10 @@
           }
         }
       }
+    },
+    "lock": {
+      "description": "Whether to use a lock file or the path to use for the lock file. Can be overridden by CLI arguments.",
+      "type": ["string", "boolean"]
     }
```

Comment:
Maybe we could add a `"default": true` here?

Reply:
Done!